src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -559,6 +559,12 @@ static void ConfigureFeatureFlags()
 #if __ANDROID__
 			Uno.WinRTFeatureConfiguration.StoreContext.TestMode = true;
 #endif
+
+#if IS_CI_OR_DEBUG && __SKIA__
+			// Lower the framerate so that CI agents don't slow down too much
+			// as they're running with software rendering.
+			FeatureConfiguration.CompositionTarget.FrameRate = 15;
```

Comment:
Is this needed for debugging too? Just would be nice to see the smoothness when developing 😁

Reply:
We'll adjust this for different builds, I'll make a separate PR to validate the behavior in CI.